src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -88,6 +88,36 @@ bool operator!=(const ApparentHorizon<Frame>& lhs,
 
 }  // namespace OptionHolders
 
+namespace OptionTags {
+struct ApparentHorizonGroup {
```

Comment:
Make this plural please, and you can omit the `Group` if you like: `ApparentHorizons`. Then you can also remove the (redundant) `name()` function.